lib/database_dumper.rb

```diff
@@ -1205,6 +1203,259 @@ def self.actions_to_column_sanitizers(columns_by_action)
+          id
+          format
+          name
+          rank
+        ],
+        fake_values: {
+          # Copy over column to keep backwards compatibility
+          "cell_name" => "name",
+        },
+      ),
+    }.freeze,
+    "formats" => {
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w[
+          id
+          expected_solve_count
+          name
+          sort_by
+          sort_by_second
+          trim_fastest_n
+          trim_slowest_n
+        ],
+      ),
+    }.freeze,
+    "countries" => {
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w[
+          id
+          iso2
+          name
+        ],
+        fake_values: %w[
+          continent_id
+        ],
+      ),
+    }.freeze,
+    "continents" => {
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w[
+          id
+          latitude
+          longitude
+          name
+          zoom
+        ],
+        fake_values: %w[
+          record_name
+        ],
+      ),
+    }.freeze,
+    "persons" => {
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w[
+          name
+          gender
+        ],
+        fake_values: {
+          "id" => "wca_id",
+          "sub_id" => "sub_id",
+          "country_id" => "country_id",
+        },
+      ),
+    }.freeze,
+    "competitions" => {
+      where_clause: PUBLIC_COMPETITION_JOIN,
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w[
+          id
+          name
+          information
+          external_website
+          venue
+          latitude
+          longitude
+        ],
+        fake_values: {
+          "city_name" => "city_name",
+          "country_id" => "country_id",
+          "venue_address" => "venue_address",
+          "venue_details" => "venue_details",
+          "cell_name" => "cell_name",
+          "cancelled" => "(competitions.cancelled_at IS NOT NULL AND competitions.cancelled_by IS NOT NULL)",
+          "event_specs" => "REPLACE(GROUP_CONCAT(DISTINCT competition_events.event_id), \",\", \" \")",
+          "delegates" => "GROUP_CONCAT(DISTINCT(CONCAT(\"[{\", users_delegates.name, \"}{mailto:\", users_delegates.email, \"}]\")) SEPARATOR \" \")",
+          "organizers" => "GROUP_CONCAT(DISTINCT(CONCAT(\"[{\", users_organizers.name, \"}{mailto:\", users_organizers.email, \"}]\")) SEPARATOR \" \")",
+          "year" => "YEAR(start_date)",
+          "month" => "MONTH(start_date)",
+          "day" => "DAY(start_date)",
+          "end_month" => "MONTH(end_date)",
+          "end_day" => "DAY(end_date)",
+        }.freeze,
+      ),
+      tsv_sanitizers: actions_to_column_sanitizers(
+        fake_values: {
+          "information" => "REGEXP_REPLACE(information, '[[:space:]]+', ' ')",
+        },
+      ),
+    }.freeze,
+    "scrambles" => {
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w[
+          scramble
+        ],
+        fake_values: %w[
```

Comment:
`fake_values` without actually telling what should be faked. How was this not caught during testing?